pyproject.toml

```
@@ -111,6 +111,16 @@ omit= [
   "data_safe_haven/resources/*",
 ]
 
+[tool.hatch.build.targets.sdist]
+exclude = [
+  "docs/",      # documentation source
+  "tests/",     # package tests
+  "typings/",   # mypy configuration
+  ".*",         # configuration dotfiles
+  "CODEOWNERS", # GitHub configuration
+  "*.md",       # GitHub configuration
+]
+
```

I think it would be better to explicitly only include what we want.
I think those exclude rules would still package untracked files which don't match any of those globs.

```suggestion
[tool.hatch.build.targets.sdist]
include_only = [
  "data_safe_haven",
  "README.md",
  "CITATION.cff",
  "DISCLAIMER.md",
  "LICENSE",
  "SECURITY.md",
]

```

---

Or,

```suggestion
[tool.hatch.build.targets.sdist]
include = [
  "data_safe_haven/",
  "README.md",
  "CITATION.cff",
  "DISCLAIMER.md",
  "LICENSE",
  "SECURITY.md",
]
exclude = [
  "*.pyc",
]
```

?

---

Switched to include-based method in c5801b8